src/example/pkg/example.go

```diff
@@ -0,0 +1,22 @@
+package pkg
+
+import (
+	"fmt"
+
+	"github.com/danielgtaylor/unistyle" // this is a third-party package
```

Comment:
Not sure if this comment was added automatically or not, but if it wasn't, let's capitalize to match our usual style:
```suggestion
	"github.com/danielgtaylor/unistyle" // This is a third-party package
```

Reply:
No I added it

Reply:
This conversation was resolved accepting the suggestion or explaining why it was resolved, so I unresolved it.